ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -115,6 +115,7 @@ - (void)layoutSubviews
 
 - (void)setAnnotations:(NSMutableArray *)annotations
 {
+		[self removeAllAnnotations];
```

Comment:
This has already been fixed in https://github.com/mapbox/react-native-mapbox-gl/pull/225